Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/burrower.yml

```diff
@@ -31,17 +31,23 @@
     - ActionXenoTailStab
     - ActionXenoAcidNormal
     - ActionXenoOrderConstruction
-    #    - ActionXenoDigTunnel
     - ActionXenoPlantWeeds
-    #    - ActionXenoBurrow
-    #    - ActionXenoTremor
+    - ActionXenoTremor
     - ActionXenoDevolve
     tier: 2
     hudOffset: 0,0.56
     unlockAt: 420 # 7 minutes
+  - type: XenoConstruction
+    canBuild:
+    - WallXenoResin
+    - WallXenoMembrane
+    - DoorXenoResin
```

Comment:
The Burrower cannot secrete resin so you don't need these 3
```suggestion
```

Reply:
Currently burrower evolution is NOT ready and wont be in the game, i will delete this part when its done, added only when debuging (just i was using this to look at Tunnel construction, ability was an easy way to spawn weeds)